src/backoffice/dashboards/performance-profiling/dashboard-performance-profiling.element.ts

```diff
@@ -1,15 +1,85 @@
 import { UUITextStyles } from '@umbraco-ui/uui-css/lib';
 import { css, html, LitElement } from 'lit';
-import { customElement } from 'lit/decorators.js';
+import { customElement, state } from 'lit/decorators.js';
+
+import { ApiError, ProblemDetails, ProfilingResource } from '@umbraco-cms/backend-api';
+import { UmbContextConsumerMixin } from '@umbraco-cms/context-api';
+import { UmbNotificationDefaultData, UmbNotificationService } from '@umbraco-cms/services';
 
 @customElement('umb-dashboard-performance-profiling')
-export class UmbDashboardPerformanceProfilingElement extends LitElement {
-	static styles = [UUITextStyles, css``];
+export class UmbDashboardPerformanceProfilingElement extends UmbContextConsumerMixin(LitElement) {
+	static styles = [
+		UUITextStyles,
+		css`
+			uui-toggle {
+				font-weight: bold;
+			}
+
+			h4 {
+				margin-bottom: 0;
+			}
+
+			h4 + p {
+				margin-top: 0;
+			}
+		`,
+	];
+
+	@state()
+	private _profilingStatus = false;
+
+	private _notificationService?: UmbNotificationService;
+
+	private async _getProfilingStatus() {
+		try {
+			const status = await ProfilingResource.getProfilingStatus();
+			status.enabled === true ? (this._profilingStatus = status.enabled) : (this._profilingStatus = false);
+		} catch (e) {
+			if (e instanceof ApiError) {
+				const error = e as ProblemDetails;
+				const data: UmbNotificationDefaultData = { message: error.message ?? 'Something went wrong' };
+				this._notificationService?.peek('danger', { data });
+			}
+		}
+	}
+
+	constructor() {
+		super();
+		this._getProfilingStatus();
+		this.consumeAllContexts(['umbNotificationService'], (instances) => {
+			this._notificationService = instances['umbNotificationService'];
+		});
+	}
 
 	render() {
 		return html`
 			<uui-box>
 				<h1>Performance Profiling</h1>
+				<p>
```

Comment:
This text should only be shown if the profiler is actually enabled. If not, there is another text to show if you check "profiler.html":

```html
                    <p>
                        Umbraco currently does not run in debug mode, so you can't use the built-in profiler. This is how it should be for a production site.
                    </p>
                    <p>
                        Debug mode is turned on by setting <b>debug="true"</b> on the <b>&lt;compilation /&gt;</b> element in web.config.
                    </p>
```